Monika After Story/game/script-topics.rpy

```diff
@@ -12818,6 +12818,42 @@ label monika_enjoyingspring:
         m 2rkc "...but I guess there's no real way to avoid it, is there?"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_life_skills",category=['advice','life'],prompt="Life skills",random=True))
+
+label monika_life_skills:
+    m 1euc "You know, [player]..."
+    m 1rud "I was just thinking about what I got out of high school."
+    m 2rksdlb "You would think that with all the things I had going for me, that I would be pretty set for the future."
+    m 3rksdla "But when I think about it, despite all that, I'm not sure how many life skills I learned."
+    m 1eka "Sure, I was on top of all my classes, and I guess I learned a lot of interesting things..."
+    m 1euc "When it comes to life skills though, I can't really think of many classes I learned from."
+    m "I don't think I'm the only one in that regard."
+    m 1hksdlb "I've heard of some people graduating and then falling apart because they don't know how to do taxes or book appointments."
+    m 1rksdlc "I feel like schools don't really do a great job teaching life skills."
```

Comment:
```suggestion
    m 3ekc "I've heard of some people graduating and then falling apart because they don't know how to do taxes or book appointments."
```